src/latbin/lattice-compose.cc

```diff
@@ -121,7 +164,23 @@ int main(int argc, char *argv[]) {
         }
       }
       delete fst2;
-    } else {
+    }
```

Comment:
Thanks!

Would you mind formatting the else in a more standard way that won't upset auto-formatters?
E.g.
```
} else {
  /* comment here */
}
```
.. I see why you did what you did, but let's be consistent with other code.
Are you confident that this won't break existing code?